feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -8,34 +8,58 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringArrayResource
 import androidx.compose.ui.res.stringResource
 import androidx.core.text.isDigitsOnly
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.iteneum.RepairModel
 import com.iteneum.designsystem.components.*
 import com.iteneum.designsystem.theme.LPTypography
 import com.iteneum.designsystem.theme.LeasePertTheme
 import com.iteneum.repair.R
+import com.iteneum.repair.data.RepairViewModel
 
 /**
  * Function that creates [RepairView] compose. for user's service request.
  * User will require to fill all the correspondent fields on the screen.
  *
+ * Function [RepairView] has
+ * @param navigateToApartment - to get the function to navigate ApartmentView when clicking button
+ * @param repairViewModel - to get the Repair View Model access
+ *
+ * Function [RepairContainer] has
+ * @param dataInfo - to use if for displaying Unit in the Unit Text Field
+ * @param repairViewModel -  to get the Repair View Model access
+ * @param navigateToApartment - its passed from RepairView() for same purpose
+ *
  * @author Jose Miguel Garcia Reyes
  */
 
+@Composable
+fun RepairView(
+    navigateToApartment: () -> Unit,
+    repairViewModel: RepairViewModel = hiltViewModel()
+) {
+    LaunchedEffect(true) {
+        repairViewModel.getInformation()
+    }
+    RepairContainer(
+        dataInfo = repairViewModel.dataInfo.value,
+        navigateToApartment = navigateToApartment
+    )
+}
+
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun RepairView() {
+fun RepairContainer(
```

Comment:
Could you give me a structure example? Because since I am using compose I thought that the observers were this methods on file "RepairViewModel"

![image](https://github.com/irv205/droid_leaspert/assets/25538206/4c913300-9179-4ec0-ab3c-4bf1b053e0a2)


Or do you want something like this other example like when using Live Data for observables?
![image](https://github.com/irv205/droid_leaspert/assets/25538206/319f60ab-a4dd-4e1e-be9e-c300b4ebec32)

I don't know a lot of this yet, so your comments are really helpful. Thank you!

Reply:
What is the status?

Reply:
Not sure about this one. I thought that _remember_ is not required anymore since the "ViewModel" is the one that handles that data. As I understood, the View does not require to remember any value. @irv205 
Therefore, I do not see the necessity of adding remember here.

Reply:
ViewModel reworked with events. Aided by Carlos. Done!